src/components/v2/V2Create/DeployProjectButton.tsx

```diff
@@ -80,7 +84,7 @@ export default function DeployProjectButton() {
     <>
       <Button
         onClick={() => setDeployProjectModalVisible(true)}
-        type="primary"
+        type={notPrimary ? 'default' : 'primary'}
```

Comment:
**Suggestion:** maybe we could pass in a `buttonType` prop and just use it in all cases (`type={buttonType}`)

Reply:
<img width="663" alt="Screen Shot 2022-02-27 at 11 28 04 pm" src="https://user-images.githubusercontent.com/22532527/155882334-b328ccfc-218f-4cfd-bf8b-fd9d1a9daeb1.png">
<img width="657" alt="Screen Shot 2022-02-27 at 11 29 55 pm" src="https://user-images.githubusercontent.com/22532527/155882388-49ecd1ba-cbb3-42e5-811b-1c74b6758e6a.png">


great idea! however it looks like i have some problem passing through 'type', do you know why?

Reply:
Where you give the type of `type` (in the component function arguments), you must use specify all or some values of the ButtonType options. i.e. instead of going `type: string` you'd have to go `type: "default" | "primary"`

Reply:
you should be able to get the `ButtonType` type exported from antd and use that instead of simply `string`

also, i believe `'default'` will be used as the type if no type is passed, or type === undefined

<img width="314" alt="image" src="https://user-images.githubusercontent.com/79433522/155916123-67d49255-2350-4f77-a134-a750dd2fae44.png">